pages/statistics.jsx

```diff
@@ -0,0 +1,28 @@
+import Header from "@/components/Header";
+import Stat from "@/components/Stat";
+import styles from "@/styles/pages/statistics.module.scss";
+
+export default function Statistics() {
+	const data = [
```

Comment:
This is hardcoded and never going to change - you should define it outside function so it would defined once.
Also, in javascript, there is a naming convention for immutable constants to be CONSTANT_CASE.
p.s not all constants are immutable - arrays for example, I can push to them.

* This is a convention, there might be places where it's not applied and it's not a hard rule.
Following a convention is for consistency and clarity sake.

Reply:
noted